pywinauto/unittests/test_application.py

```diff
@@ -1189,22 +1188,70 @@ def test_folder_list(self):
                               u'CtrlTest.exe', u'mfc100u.dll', u'RebarTest.exe', u'RowList.exe', u'TrayMenu.exe'])
             self.assertEqual(files_list.item('RebarTest.exe').window_text(), 'RebarTest.exe')
 
+        def test_get_list_of_windows_uia(self):
+            """Test that method return list of windows"""
+            dlgs = self.desktop.windows()
+            self.assertTrue(len(dlgs) > 1)
+
+        def test_set_backend_to_windows_uia(self):
+            """Set backend to method windows, except exception ValueError"""
+            with self.assertRaises(ValueError):
+                self.desktop.windows(backend='win32')
+            with self.assertRaises(ValueError):
+                self.desktop.windows(backend='uia')

```

Comment:
I know this is not a subject, but similar checks are useful for method `window()` as well.

Reply:
Okay, I will add it to the method window() too.